### ISSUE TITLE
Input fields on homepage are missing labels

### ISSUE THREAD

**Issue by schalkneethling** [mdn/mdn-fiori#81]
The search input in the header, as well as the input field for the newsletter, does not have an associated label.

## Bugzilla

https://bugzilla.mozilla.org/show_bug.cgi?id=1563464
